README.md

```diff
@@ -27,15 +27,15 @@ Upload `brakeman.hpi`.
 Add/append a shell command to your job configuration like (this is just an example):
 
     gem install brakeman --no-rdoc --no-ri &&
-    brakeman -o brakeman-output.tabs
+    brakeman -o brakeman-output.json
 
 Or, if [rvm](https://rvm.beginrescueend.com/) is available, you can use something like:
 
-    bash -l -c 'rvm use 1.8.7 &&
+    bash -l -c 'rvm use 2.3.1 &&
```

Comment:
I think the latest and greatest is 2.4.1 now?

(Not that it matters, have several projects stuck on 2.1 at the day job)